plug-ins/clan/command/cclan.cpp

```diff
@@ -1032,12 +1032,12 @@ void CClan::clanMember( PCharacter *pc, DLString& argument )
         clanMemberHelp( pc );
         return;
     }
-
+/*
```

Comment:
Can safely remove the whole block. Also adjust 'clan member help' function to remove 'leaders' mention.